### ISSUE TITLE
Unicode character � not supported in SC Admin Panel Query

### ISSUE THREAD

**Issue by atifwaqar** [Starcounter/Home#79]
SC Version : 2.3.0.5916
Query : 
```sql
select *
from adfenix.database.classifiedad 
where displaynameclassifiedad like '%�%'
```

StackTrace : 
```
ScErrSQLIncorrectSyntax (SCERR7021): Incorrect SQL syntax. Unterminated quoted string. The error near or at position 80, near or at token: '%�%' In query: select * from adfenix.database.classifiedad where displaynameclassifiedad like '%�%'
 Version: 2.3.0.5916.
 Help page: https://github.com/Starcounter/Starcounter/wiki/SCERR7021. Help. View the log for more details.
```

![2017-04-06](https://cloud.githubusercontent.com/assets/4388707/24750003/84ad670c-1ac5-11e7-9d8c-cc037c632697.png)

**Comment by mtornwall** [comment]
@k-rus

**Comment by k-rus** [comment]
@andan342 
Can you look how it works in the native parser? The error seems to be from the native parser.

**Comment by k-rus** [comment]
Note it is reported for 2.3.

**Comment by andan342** [comment]
What is the Unicode code point for that character? Or its UTF8 or UTF16?

I tried to reproduce with basic and supplementary Unicode examples - the query is syntactically correct in **v2.3**.

**Comment by k-rus** [comment]
> What is the Unicode code point for that character? Or its UTF8 or UTF16 binary representation?

cc @atifwaqar

**Comment by atifwaqar** [comment]
� is UTF8. I checked it from [here](https://en.wikipedia.org/wiki/Specials_(Unicode_block))

**Comment by k-rus** [comment]
@andan342 I followed OP and reproduced the issue, i.e., by calling a query similar to OP from Administrator.

Steps to reproduce:

1. Obtain Level0 for 2.3 and build it
2. Obtain Level1 for 2.3 and build it
3. Install Starcounter or set up Personal server or make sure the server is correctly referenced from configuration
4. Start a database, e.g., `staradmin start db`
5. Open Administrator
6. Submit a query similar to the OP, e.g., `select * from rawview where name like '%�%'`

**Comment by andan342** [comment]
I confirm the error is reproducible via *Starcounter Personal Administrator* web interface.

**Comment by atifwaqar** [comment]
Any update on this one?

**Comment by k-rus** [comment]
> Any update on this one?

@andan342 was away. He will look to the issue this week I believe. How is it serious for you, emergent to fix?

**Comment by atifwaqar** [comment]
It is kind of serious, we have workarounds for this issue, but workarounds are not good when we are coding new features.

**Comment by k-rus** [comment]
> It is kind of serious, we have workarounds for this issue, but workarounds are not good when we are coding new features.

Thank you for letting us to know. We will prioritise it.

**Comment by andan342** [comment]
Fixed in **v2.3** and **develop**.
